modules/@apostrophecms/import-export-doc-type/index.js

```diff
@@ -27,5 +27,27 @@ module.exports = {
     }
 
     self.apos.doc.addContextOperation(operation);
+
+    // Add import operation for singleton doc types
+    if (self.options.singleton === true || !!self.options.singletonAuto) {
+      if (self.options.importExport?.import === false) {
+        return;
+      }
+      console.log('self.options.singleton', self.name, (self.options.singleton === true || !!self.options.singletonAuto), (self.options.singleton === true || self.options.singletonAuto));
+      const importOperation = {
+        action: 'import-export-import',
+        context: 'update',
+        label: 'aposImportExport:import',
+        modal: 'AposImportModal',
+        props: {
+          action: 'import-export-import'
+        }
+      };
+      if (excludedTypes.length) {
+        operation.if = { $and: excludedTypes };
+      }
```

Comment:
This `excludedTypes` are built on the `export` option, it should be a specific one for `import` option.
(we can exclude export and import independently).